public/app/core/components/AppChrome/MegaMenu/MegaMenu.tsx

```diff
@@ -76,9 +76,11 @@ export const MegaMenu = memo(
       }
 
       // refocus on undock/menu open button when changing state
-      setTimeout(() => {
-        document.getElementById(state.megaMenuDocked ? 'mega-menu-toggle' : 'dock-menu-button')?.focus();
-      });
+      if (!config.featureToggles.singleTopNav) {
+        setTimeout(() => {
+          document.getElementById(state.megaMenuDocked ? 'mega-menu-toggle' : 'dock-menu-button')?.focus();
```

Comment:
Is there a specific reason why you are not using MEGA_MENU_TOGGLE_ID and DOCK_MENU_BUTTON_ID here? If not, please add both.

Reply:
only that it's gonna be removed soon anyway 😄 i'll change it...